### PR TITLE
Add test cases to ImportCommandTest

### LINKED ISSUE #205: Add additional test cases for v1.4

### PR DESCRIPTION
**Pull request by samuelcheongws**

### CODE REVIEW

File: docs/team/samuelcheongws.md [hunk 1]
```diff
@@ -19,13 +19,17 @@ Given below are my contributions to the project.
...
 * Added `ApplicationStatusTest` and `SortCommandTest` and other Import command tests to increase code coverage and ensure that programs work as expected.
 
-* **Documentation**: `to be added soon`
+* **Documentation**: 
   * User Guide: Added documentation for the features import and edit
   * Developer Guide: Added documentation, sequence diagrams, and activity diagrams for the features import and edit
 
 * **Community**:
-* * Reviewed several PRs, including https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/92.
+* Reviewed several PRs, including [#97](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/92)
```
Comment (benjytan45678) [inline docs/team/samuelcheongws.md]: You might want to double check the link with [#97}

File: docs/team/samuelcheongws.md [hunk 2]
```diff
@@ -19,13 +19,17 @@ Given below are my contributions to the project.
...
 * Found several bugs for CS2103T-W12-3 and suggested improvements.
+* Reported bugs for another team [here](https://github.com/samuelcheongws/ped/issues)
```
Comment (benjytan45678) [inline docs/team/samuelcheongws.md]: Might want to specify other team grp number?

File: docs/team/samuelcheongws.md [hunk 3]
```diff
@@ -19,13 +19,17 @@ Given below are my contributions to the project.
 
 * **Code contributed**: Refer to my [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=w10-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=zoom&zA=samuelcheongws&zR=AY2223S1-CS2103T-W10-3%2Ftp%5Bmaster%5D&zACS=134.73076923076923&zS=2022-09-16&zFS=w10-3&zU=2022-11-04&zMG=false&zFTF=commit&zFGS=groupByRepos&zFR=false)
 
+* **Project management**:
+  * Managed releases `v1.4` on GitHub
+
 * **Enhancements to existing features**: 
 * Added `ApplicationStatusTest` and `SortCommandTest` and other Import command tests to increase code coverage and ensure that programs work as expected.
```
Comment (ShummyOwnzYou) [inline docs/team/samuelcheongws.md]: Good to add the relevant PRs and link them here. Same goes for your import command feature.

File: src/test/java/seedu/trackascholar/logic/commands/ImportCommandTest.java [hunk 4]
```diff
@@ -30,6 +31,24 @@ public void constructor_nullApplicant_throwsAssertionError() {
...
+    public void execute_noFile_throwsCommandException() {
+        Path importedFilePath =
+                Paths.get("src/test/data/JsonImportCommandTest", "fileWithWrongName.json");
+        ImportCommand importCommand = new ImportCommand(ImportCommand.REPLACE, importedFilePath);
+
+        assertCommandFailure(importCommand, model, ImportCommand.MESSAGE_NO_FILE_FOUND_ERROR);
+    }
+
+    @Test
+    public void execute_invalidFileFormat_throwsCommandException() {
```
Comment (ShummyOwnzYou) [inline src/test/java/seedu/trackascholar/logic/commands/ImportCommandTest.java:44]: Good job handling exception cases!

**Review by ShummyOwnzYou — COMMENTED** [review]
LGTM, apart from some minor areas of improvement.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -720,9 +720,19 @@ testers are expected to do more *exploratory* testing.
...
+      1. Run TrackAScholar (make some changes to the file) and close it
+      2. It will create a new ./data/ folder with the trackAScholar.json file in the same folder 
+      3. Insert the trackAScholarImport.json file here (Note that the name needs to be exactly as stated)
+   1. Test case: `import k`<br>
+      Expected: trackAScholarImport has been imported and new applicants are displayed int the GUI. Applicants with the same name not be imported. A successful import message is displayed.
+
+   1. Test case: `import r`<br>
+      Expected: trackAScholarImport has been imported and new applicants are displayed int the GUI. Applicants with the same name will be replaced by the applicant with updated fields from trackAScholarImport. A successful import message is displayed.
+
+   1. Incorrect delete commands to try: `import`, `import x`, `...` (where x is not r or k)<br>
```
Comment (benjytan45678) [inline docs/DeveloperGuide.md]: incorrect ***import*** commands

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -720,9 +720,19 @@ testers are expected to do more *exploratory* testing.
...
-1. Run TrackAScholar (make some changes to the file) and close it
-2. It will create a new ./data/ folder with the trackAScholar.json file in the same folder 
-3. Insert the trackAScholarImport.json file here (Note that the name needs to be exactly as stated)
+1. Importing a trackAScholarImport file into the current model
+   1. Setup and Prerequisites:
+      1. Run TrackAScholar (make some changes to the file) and close it
+      2. It will create a new ./data/ folder with the trackAScholar.json file in the same folder 
+      3. Insert the trackAScholarImport.json file here (Note that the name needs to be exactly as stated)
+   1. Test case: `import k`<br>
+      Expected: trackAScholarImport has been imported and new applicants are displayed int the GUI. Applicants with the same name not be imported. A successful import message is displayed.
```
Comment (benjytan45678) [inline docs/DeveloperGuide.md]: are displayed ***in***?

Comment (benjytan45678) [inline docs/DeveloperGuide.md]: Applicants with the same name ***will*** not

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -720,9 +720,19 @@ testers are expected to do more *exploratory* testing.
...
+
+   1. Test case: `import r`<br>
+      Expected: trackAScholarImport has been imported and new applicants are displayed int the GUI. Applicants with the same name will be replaced by the applicant with updated fields from trackAScholarImport. A successful import message is displayed.
```
Comment (benjytan45678) [inline docs/DeveloperGuide.md]: Same error as previous

**Review by benjytan45678 — COMMENTED** [review]
LGTM, merged

**Pull request merged**